components/nexusoperations/executors_test.go

```diff
@@ -311,7 +360,17 @@ func TestProcessInvocationTask(t *testing.T) {
 			ctrl := gomock.NewController(t)
 			listenAddr := nexustest.AllocListenAddress(t)
 			h := handler{}
-			h.OnStartOperation = tc.onStartOperation
+			h.OnStartOperation = func(
+				ctx context.Context,
+				service, operation string,
+				input *nexus.LazyValue,
+				options nexus.StartOperationOptions,
+			) (nexus.HandlerStartOperationResult[any], error) {
+				if tc.checkStartOperationOptions != nil {
+					tc.checkStartOperationOptions(t, options)
+				}
```

Comment:
nit: you could have just merged this into `onStartOperation`. Not blocking the PR though.